### PR TITLE
Support --keep-going with BuildDoc setup command

### PR DESCRIPTION
**Pull request by blueyed**
This allows for `tox -e docs -- -n -W --keep-going`.

### CODE REVIEW

File: sphinx/locale/sphinx.pot [hunk 1]
```diff
@@ -1252,7 +1252,7 @@ msgid "turn warnings into errors"
 msgstr ""
 
 #: sphinx/cmd/build.py:189
-msgid "With -W, Keep going when getting warnings"
+msgid "With -W, keep going when getting warnings"
```
Comment (tk0miya) [inline sphinx/locale/sphinx.pot]: `sphinx.pot` will be updated by tooling. So you don't need to touch this.

**Review by tk0miya — APPROVED** [review]
LGTM with nits

File: sphinx/locale/sphinx.pot (hunk 1 above)
Comment (blueyed) [inline sphinx/locale/sphinx.pot]:
```suggestion
msgid "With -W, Keep going when getting warnings"
```

**Comment by blueyed** [comment]
Ok. Please squash-merge then.

**Comment by tk0miya** [comment]
Thank you for quick work! Merged!

**Pull request merged**